components/docker-up/docker-up/main.go

```diff
@@ -71,6 +72,8 @@ func main() {
 	pflag.BoolVarP(&opts.Verbose, "verbose", "v", false, "enables verbose logging")
 	pflag.BoolVar(&opts.RuncFacade, "runc-facade", true, "enables the runc-facade to handle rootless idiosyncrasies")
 	pflag.StringVar(&opts.BinDir, "bin-dir", filepath.Dir(self), "directory where runc-facade is found")
+	pflag.StringVar(&opts.ContainerIf, "iface", "ceth0", "container iface name")
+	pflag.StringVar(&opts.DataRoot, "data-root", "/workspace/.docker-root", "dockerd data root folder")
```

Comment:
We discussed with @iQQBot try not to change it to stay close to the real workspace.

Instead we are going to mount in gp rebuild on top of the workspace mount.